code/controllers/UserController.php

```diff
@@ -66,27 +67,35 @@ public function sendInvite($data, Form $form)
         }
 
         $invite = UserInvitation::create();
-        $form->saveInto($invite);
-        try {
-            $invite->write();
-        } catch (ValidationException $e) {
+        if ($invite->canCreate(Member::CurrentUser()) == true){
```

Comment:
You're right. 
Would you agree with me that by implementing the changes securing the `UserController::index()` action above we're eliminating this problem, making this extra check unneeded?